python/ray/data/dataset_pipeline.py

```diff
@@ -63,7 +63,15 @@
 ]
 
 # Operations that operate over the stream of output batches from the pipeline.
-_OUTPUT_ITER_OPS = ["take", "take_all", "show", "to_tf", "to_torch"]
+_OUTPUT_ITER_OPS = [
```

Comment:
nit: given https://github.com/ray-project/ray/pull/26394 is merged, could you rebase and define `iter_torch_batches` and `iter_tf_batches` explicitly in `DatasetPipeline`? Thanks.

I can clean up all these methods to extrac common logic into a utility class later per https://github.com/ray-project/ray/pull/26394#discussion_r917120687.

Reply:
Yep I have this rebased locally, will push up in a bit!